src/os/bluestore/BlueStore.cc

```diff
@@ -4220,7 +4220,7 @@ int BlueStore::_open_fm(bool create)
         }
         assert(start + l + u <= end);
 
-	dout(20) << "  free 0x" << std::hex << start << "~" << l
+	dout(20) << __func__ << "  free 0x" << std::hex << start << "~" << l
```

Comment:
Nit: there is an unwanted space (2 spaces) before `free 0x`. Can you fix it along with these changes ?

Reply:
will update.